doc/_admin-guide/080_Log/030_Filters/002_Comparing_macro_values.md

```diff
@@ -103,11 +103,13 @@ Note that:
 
 The following operators are available:
 
-|  Numerical operator   |String operator  | Meaning|
-|--------------------|-----------------|----------------------|
-|  ==  |                 eq |               Equals|
-|  !=   |                ne  |              Not equal to|
-| \>    |               gt   |             Greater than|
-| \<     |              lt    |            Less than|
-| \>=     |             ge     |           Greater than or equal|
-| =\<      |            le      |          Less than or equal|
+| Numerical Operator | String Operator | Meaning                          |
+|--------------------|-----------------|----------------------------------|
+| ==                 | eq              | Equals                           |
+| ===                |                 | Equals and same type             |
+| !=                 | ne              | Not equal to                     |
+| !==                |                 | Not equal to and same type       |
+| >                  | gt              | Greater than                     |
+| <                  | lt              | Less than                        |
+| >=                 | ge              | Greater than or equal to         |
+| <=                 | le              | Less than or equal to            |
```

Comment:
Won't the unescaped `<` and `>` signs cause any problems?

Reply:
it seems to be not

<img width="899" height="389" alt="Screenshot 2025-08-22 at 12 02 26" src="https://github.com/user-attachments/assets/52ec2c19-c1f5-48e6-a53d-ee07b9b26b20" />